src/imitation/algorithms/bc.py

```diff
@@ -260,7 +260,7 @@ def __init__(
         self.l2_weight = l2_weight
 
     @property
-    def policy(self) -> policies.BasePolicy:
+    def policy(self) -> policies.ActorCriticPolicy:
```

Comment:
Ah, you're right we need it to be an `ActorCriticPolicy` as we call `evaluate_actions`. Can you update other type annotations? `__init__` takes a `policy: Optional[policies.BasePolicy] = None` for example, we might still use `BasePolicy` elsewhere as well.